src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -102,4 +102,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       Include="$(MSBuildThisFileDirectory)..\codestyle\vb\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.dll"
       IsImplicitlyDefined="true" />
   </ItemGroup>
+  
+  <!-- Build Properties Exposed to CodeStyle Analyzers -->
+  <ItemGroup>
```

Comment:
I think we want to condition the item group on this being C# or VB language.

Reply:
yep I think we want this: `<ItemGroup Condition="'$(Language)' == 'C#' Or '$(Language)' == 'VB'">`

Reply:
Added the condition to the ItemGroup